doc/build_apps/get_started.rst

```diff
@@ -0,0 +1,172 @@
+Quick Start
+===========
+
+Application Development using CCF Overview
+-------------------------------------------
+
+-  :ref:`What is Confidential Consortium Framework (CCF) <overview/what_is_ccf:What is CCF?>`
+-  Read the :doc:`CCF overview </overview/index>` and get familiar with :ref:`overview/what_is_ccf:Core Concepts` and `Azure confidential computing <https://learn.microsoft.com/en-us/azure/confidential-computing/>`__
+-  :doc:`Build new CCF applications </build_apps/index>` in TypeScript/JavaScript or C++
+-  CCF `Modules API reference <https://microsoft.github.io/CCF/main/js/ccf-app/modules.html>`__
+-  CCF application get started repos `CCF application template <https://github.com/microsoft/ccf-app-template>`__ and  `CCF application samples <https://github.com/microsoft/ccf-app-samples>`__
+
+Development environment
+-----------------------
+
+-  VS Code Dev Container |Open in VSCode|
+-  Github codespace: |Github codespace|
+-  Virtual Machine (`Creating a Virtual Machine in Azure to run CCF <https://github.com/microsoft/CCF/blob/main/getting_started/azure_vm/README.md>`__)
+
+(JavaScript/Typescript) Applications
+------------------------------------
+
+CCF apps can be written in JavaScript or Typescript, To test a JS/TS CCF application you need go through the following steps:
+
+-  :doc:`Start a CCF Network with at least one node </operations/start_network>`
+-  Initialize the CCF network with at least one (active member - user) :ref:`governance/open_network:Opening a Network`,
+   this can be done through :ref:`Proposals <governance/proposals:Summary>`.
+-  Create an application :ref:`deployment proposal <build_apps/js_app_bundle:Deployment>`
+-  Submit the app deployment proposal to the network and all members accept it through voting. This is a part of :ref:`Network Governance <governance/proposals:Submitting a New Proposal>`.
+-  :doc:`Open the CCF network for users </governance/open_network>`
+-  Start testing of your application endpoints
+
+Build Application
+~~~~~~~~~~~~~~~~~
+
+The application building prerequisites [:doc:`CCF </build_apps/install_bin>`, `NodeJS <https://nodejs.org>`__ and `NPM <https://www.npmjs.com>`__] must be installed, all will be preinstalled if you are using devcontainer environment, otherwise you need to install them manually.
```

Comment:
```suggestion
The application building prerequisites [:doc:`CCF </build_apps/install_bin>`, `NodeJS <https://nodejs.org>`__ and `NPM <https://www.npmjs.com>`__] must be installed, all will be preinstalled if you are using the devcontainer environment, otherwise you need to install them manually.
```